company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise7Test.java

```diff
@@ -40,7 +47,9 @@
     @Test
     public void customersByCity()
     {
-        MutableListMultimap<String, Customer> multimap = null;
+        MutableListMultimap<String, Customer> multimap = new FastListMultimap<>() ;
```

Comment:
This could be replaced with this API (as hinted in this class above) - MutableList#groupBy(Function)
MutableList in this case will be `this.company.getCustomers()` and groupBy can be applied to `Customer::getCity`